packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -104,15 +104,51 @@ This component renders a list of suggestions coming from the user queries histor
 
 No props are required for the usage of this component.
 
-```vue
-<HistoryQueries />
+```vue live
+<template>
+  <SearchInput />
+  <HistoryQueries />
+</template>
+
+<script>
+  import { SearchInput } from '@empathyco/x-components/search-box';
+  import { HistoryQueries } from '@empathyco/x-components/history-queries';
+
+  export default {
+    name: 'HistoryQueriesDemo',
+    components: {
+      SearchInput,
+      HistoryQueries
+    }
+  };
+</script>
 ```
 
 The component has two optional props. `animation` to render the component with an animation and
 `maxItemToRender` to limit the number of history queries will be rendered (by default it is 5).
 
-```vue
-<HistoryQueries :animation="FadeAndSlide" :maxItemsToRender="10" />
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a `div` for the code to work